src/Satooshi/Bundle/CoverallsV1Bundle/Command/CoverallsV1JobsCommand.php

```diff
@@ -83,6 +83,13 @@ protected function configure()
             array()
         )
         ->addOption(
+            'json_path',
+            '-o',
+            InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
```

Comment:
No problem, PR is nice so far and I would like to have it, but it's not emergency that we need to drop everything and focus on it ;) Thanks for your working so far!

About commit - feel free to just push new one.
Then, commits will be squashed during merging ;)

(cc @tmatsuo to do on github GUI we probably will need to change repo settings, but we could do it in console if needed - I could merge this PR when it will be ready!)